### ISSUE TITLE
Allow suppressing RuntimeException output when running scripts

### ISSUE THREAD

**Issue by jonsa** [composer/composer#5281]
With the following composer.json:

```json
{
    "require-dev": {
        "phpunit/phpunit": "^4.8",
    },
    "scripts": {
        "test": "phpunit"
    }
}
```

When I run this command:

```
composer test
```

I get this output:

```
composer test
> phpunit

...Removed phpunit output...

FAILURES!
Tests: 32, Assertions: 35, Failures: 1.
Script phpunit handling the test event returned with an error


  [RuntimeException]
  Error Output:


test [--dev] [--no-dev] [--] [<args>]...
```

And I expected this to happen:

composer not to output the ```Script phpunit handling the test event returned with an error``` part.

At the moment I work around this issue by defining my script as:
```
"scripts": {
    "test": "phpunit || php -r \"return 0;\""
}
```

But if it could be built into composer it would be better in my opinion.

**EDIT**

Realized I didn't provided the actual command and output but was mislead by the issue template. Here is the output with the ```-vvv``` flag

```
Running 1.0.3 (2016-04-29 16:30:15) with PHP 5.5.30 on Windows NT / 10.0
Reading ./composer.json
Loading config file ./composer.json
Checked CA file C:\...\composer-cacert-...pem: valid
Executing command (C:\...): git branch --no-color --no-abbrev -v
Reading C:/.../AppData/Roaming/Composer/composer.json
Loading config file C:/.../AppData/Roaming/Composer/composer.json
Reading C:\.../vendor/composer/installed.json
Reading C:/.../AppData/Roaming/Composer/vendor/composer/installed.json
> test: phpunit

...Removed phpunit output...

FAILURES!
Tests: 32, Assertions: 35, Failures: 1.
Script phpunit handling the test event returned with an error


  [RuntimeException]
  Error Output:


Exception trace:
 () at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/EventDispatcher/EventDispatcher.php:207
 Composer\EventDispatcher\EventDispatcher->doDispatch() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/EventDispatcher/EventDispatcher.php:94
 Composer\EventDispatcher\EventDispatcher->dispatchScript() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Command/ScriptAliasCommand.php:59
 Composer\Command\ScriptAliasCommand->execute() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Command/Command.php:259
 Symfony\Component\Console\Command\Command->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:844
 Symfony\Component\Console\Application->doRunCommand() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:192
 Symfony\Component\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:189
 Composer\Console\Application->doRun() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/vendor/symfony/console/Application.php:123
 Symfony\Component\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Console/Application.php:99
 Composer\Console\Application->run() at phar://C:/ProgramData/ComposerSetup/bin/composer.phar/bin/composer:43
 require() at C:\ProgramData\ComposerSetup\bin\composer.phar:24

test [--dev] [--no-dev] [--] [<args>]...
```

**Comment by jonsa** [comment]
Adding this for the discussion.

My work around is doing what I expect and is removing the noise. But since I always return success composer is continuing execution when I chain the script with other scripts, which is kind of bad particularly in continues integration.. Preferably composer should stop any chained execution and return the fail code but suppress any ```RuntimeException``` output.

One way would be if composer could suppress the output by default and only show it when using one of the ```-v|vv|vvv``` flags otherwise make it suppressible in the script definition.

**Comment by Seldaek** [comment]
Makes sense to reduce the output here as it's not bringing much value, fixed in 	aaf2c06
